test/pom.xml

```diff
@@ -53,7 +53,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>2.20</version>
+      <version>2.24</version>
```

Comment:
[Changelog](https://github.com/jenkinsci/jenkins-test-harness/#224-2017-aug-02)

I am wondering if https://github.com/jenkinsci/jenkins-test-harness/pull/63 is responsible for the [extremely slow Windows builds](https://ci.jenkins.io/job/Core/job/jenkins/job/PR-2959/4/execution/node/34/log/?consoleFull) (2× time on Linux), or if this is just random bad luck. Other recent PRs seem to complete, though certainly more slowly than I would expect.